DOCS-9458-email-gmail-editorial-kt (#1144)
mulesoft/docs-connectors: email/1.3/modules/ROOT/edits/email-gmail.adoc

= Connect to Gmail with Email Connector Examples - Mule 4
:page-aliases: connectors::email/email-gmail.adoc

To connect Anypoint Connector for Email (Email Connector) to your Gmail account, enable *Less Secure Apps* in your account and configure a global element for the server connection IMAPS, POP3S, or SMTPS you want to use.

> == Enable Less Secure Apps in your Gmail Account

Reviewer: ```suggestion
== Enable Less Secure Apps in Your Gmail Account
```

> Google might block third-party mail clients if you do not enable the *Less Secure Apps* setting for your Gmail account, as Google blocks any apps that are not using security protocols that Google deems mandatory unless you allow less secure apps to access your Gmail account.

Reviewer: GLOBAL "as" > "because" if that is what you mean (translation):
```suggestion
Google might block third-party mail clients if you do not enable the *Less Secure Apps* setting for your Gmail account, because Google blocks any apps that are not using security protocols that Google deems mandatory unless you allow less secure apps to access your Gmail account.
```
Here's another wording to consider:
```suggestion
To prevent Google from blocking third-party mail clients that are not using security protocols that Google deems mandatory, enable the *Less Secure Apps* setting for your Gmail account.
```

> When you try to connect to your Gmail for the first time, you will receive an email with a link for enabling less secure apps in your Gmail account.

Reviewer: Ah! It would flow even more nicely if this sentence was first:
```suggestion
When you try to connect to your Gmail for the first time, you receive an email with a link for enabling less secure apps in your Gmail account. Be sure to do this, to prevent Google from blocking third-party mail clients that are not using security protocols that Google deems mandatory:
```
Also consider adding a caveat about audiences that have two-factor auth:
```suggestion
When you try to connect to your Gmail for the first time, you receive an email with a link for enabling less secure apps in your Gmail account. If your organization is not using two-factor authentication, be sure to do this, to prevent Google from blocking third-party mail clients that are not using security protocols that Google deems mandatory:
```

To connect Email Connector to your Gmail account using a POP3S connection, configure a global element:

. In Studio, navigate to the *Global Elements* view.
. Click *Create* to open the *Choose Global Type* view.
. In the *Filter* field, type `email`, select *Email POP3*, and click *OK*.
. In the *Email POP3* window, for the *Connection* field, select *POP3S Connection*.
. In the *General* section, enter the following values:
+
* *Host* +
`pop.gmail.com`
* *Port* +
`993`
* *User* +
`user@gmail.com`
* *Password* +
`Gmailpassword`
+
[start=6]
. In the *Trust Store Configuration* section, select the *Insecure* checkbox.
. Click *OK*.

The following screenshot shows the configuration:

.POP3S Configuration to Gmail
image::email-gmail-pop3.png[POP3S Configuration to Gmail]

In the XML editor, the configuration looks like this:

[source,xml,linenums]
----
<email:pop3-config name="Email_POP3">
    <email:pop3s-connection host="pop.gmail.com"
      port="995"  user="user@gmail.com"
      password="Gmailpassword">
        <tls:context>
            <tls:trust-store insecure="true"/>
        </tls:context>
    </email:pop3s-connection>
</email:pop3-config>
----

> == Configure a SMTP Connection

Reviewer: ```suggestion
== Configure an SMTP Connection
```

> To connect Email Connector to your Gmail account using a SMTPS connection, configure a global element:

Reviewer: ```suggestion
To connect Email Connector to your Gmail account using an SMTPS connection, configure a global element:
```

. In Studio, navigate to the *Global Elements* view.
. Click *Create* to open the *Choose Global Type* view.
. In the *Filter* field, type `email`, select *Email SMTP*, and click *OK*.
. In the *Email SMTP* window, for the *Connection* field, select *SMTPS Connection*.
. In the *General* section, enter the following values:
+
* *Host* +
`smtp.gmail.com`
* *Port* +
`465`
* *User* +
`user@gmail.com`
* *Password* +
`Gmailpassword`
+
[start=6]
. In the *Trust Store Configuration* section, select the *Insecure* checkbox.
. Click *OK*.

The following screenshot shows the configuration:

.SMTPS Configuration to Gmail
image::email-gmail-smtp.png[SMTPS Configuration to Gmail]

In the XML editor, the configuration looks like this:

[source,xml,linenums]
----
<email:smtp-config name="Email_SMTP" from="sender@gmail.com">
    <email:smtps-connection host="smtp.gmail.com"
      port="465" password="Gmailpassword"
      user="user@gmail.com">
        <tls:context enabledProtocols="TLSv1.2,SSLv3">
            <tls:trust-store insecure="true"/>
        </tls:context>
    </email:smtps-connection>
</email:smtp-config>
----

== Troubleshoot SMTPS Connection Issues

> Troubleshooting SMTPS connection issues depends on whether or not you have two-factor authentication set up for your Gmail account:

Reviewer: ```suggestion
The method that you use to troubleshoot SMTPS connection issues depends on whether or not you have two-factor authentication set up for your Gmail account:
```
Another way:
```suggestion
The method that you use to troubleshoot SMTPS connection issues depends on whether or not your Gmail account uses two-factor authentication:
```

> If you have two-factor authentication set up, generate an app-specific password and use that instead of your normal password. See https://support.google.com/accounts/answer/185833[Sign in Using App Password] for details. There is no need to enable *Less Secure Apps* in your Gmail account.

Reviewer: ```suggestion
* If your account uses two-factor authentication, generate an app-specific password and use that instead of your normal password. 
+
See https://support.google.com/accounts/answer/185833[Sign in Using App Password] for details. You do not need to enable *Less Secure Apps* in your Gmail account.
```

Reviewer: (suggested adding this caveat above)

> If you do not have two-factor authentication set up, you need to set up and enable *Less Secure Apps* in your Gmail account, and if your password does not work, go to https://accounts.google.com/DisplayUnlockCaptcha[Allow Access to Your Google Account] and follow these steps: +

Reviewer: ```suggestion
* If your Gmail account does not use two-factor authentication, set up and enable *Less Secure Apps* in your Gmail account, and if your password does not work, go to https://accounts.google.com/DisplayUnlockCaptcha[Allow Access to Your Google Account] and follow these steps: +
```
